src/modules/documentExecution/dashboard/DashboardRenderer.vue

```diff
@@ -65,14 +66,30 @@ export default defineComponent({
         ...mapState(dashboardStore, {
             dashboard: 'dashboards',
             selectedSheetIndex: 'selectedSheetIndex'
-        })
+        }),
+        backgroundStyle(): any {
+            if (!this.dashboardModel.configuration) return ''
+
+            const backgroundConfig = this.dashboardModel.configuration.background as IBackground
+            const backgroundStyle = { 'background-size': backgroundConfig?.imageBackgroundSize || '100%', 'background-position': 'center', 'background-repeat': 'no-repeat', 'min-height': '100%' }
+
+            if (backgroundConfig?.imageBackgroundUrl) backgroundStyle['background-image'] = `url('${backgroundConfig.imageBackgroundUrl}')`
+            if (backgroundConfig?.sheetsBackgroundColor) backgroundStyle['background-color'] = backgroundConfig.sheetsBackgroundColor
+            backgroundStyle['background-size'] = backgroundConfig?.imageBackgroundSize || 'contain'
+
+            return backgroundStyle
+        },
+        dashboardCss(): any {
+            return `<style>${this.dashboardModel?.configuration?.cssToRender}</style>`
+        }
     },
     mounted() {
         this.dashboardModel = this.model ?? {}
         if (!this.dashboardModel.sheets) this.dashboardModel.sheets = []
         if (this.dashboardModel.sheets.length === 0) this.dashboardModel.sheets.push({ label: 'new sheet', widgets: { lg: [] } })
-    },
 
+        console.log(this.dashboardModel.configuration.background)
```

Comment:
Could you please remove the console.log? Thank you.

Reply:
Removed